dnscommon.r2py

```diff
@@ -519,9 +520,25 @@ def _read_single_answer(answer_index, dns_query_data):
 
   # There are a lot of different types of queries. We can parse all of these ones 
   # in the same way, though.
-  simple_answers = ['A', 'NS', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR', 'MX', 'PTR']
+  simple_answers = ['A', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR']
   if answer_type in simple_answers:
     read_index, resource_data['address'] = _parse_answer_address(read_index, dns_query_data)
+  elif answer_type == 'NS':
+    read_index, resource_data['nameserver'] = _parse_address(read_index,dns_query_data)
+  elif answer_type == 'PTR':
+    read_index, resource_data['domain'] = _parse_address(read_index, dns_query_data)
+  elif answer_type == 'MX':
+    read_index+=2
+    read_index, resource_data['records'] = _parse_address(read_index, dns_query_data)
+  elif answer_type == 'AAAA':
+    read_index, resource_data['address'] = _parse_aaaa_address(read_index, dns_query_data)
+  elif answer_type == 'TXT':
+    txt_length = len(dns_query_data)-read_index
+    resource_text = []
+    for i in range(0,txt_length):
```

Comment:
As 'TXT' query contains text information like:
````
;; QUESTION SECTION:
;google.com.			IN	TXT

;; ANSWER SECTION:
google.com.		3600	IN	TXT	"v=spf1 include:_spf.google.com ip4:216.73.93.70/31 ip4:216.73.93.72/31 ~all"
````
We need controlled loop to convert raw packet to txt format and save it into answer section.